doc/source/conf.py

```diff
@@ -128,3 +138,76 @@
     "body": generate_404(),
 }
 notfound_no_urls_prefix = True
+
+
+# specify the URL of the archive here
+# Url for sphinx-design
+archive_url = "https://github.com/executablebooks/sphinx-design/tree/main/docs/snippets/rst"
```

Comment:
This variable is a constant. Hence, it should be declared it in line `209` and wiht upper casing.